games-strategy/0ad/0ad-0.27.0.ebuild

```diff
@@ -0,0 +1,234 @@
+# Copyright 2014-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_10 python3_11 python3_12 python3_13)
```

Comment:
I would start with python 3.12 as it's the stable and default version since may 2024, so older version might only add maintenance overhead.